d2l-localize-behavior.js

```diff
@@ -14,39 +14,39 @@ D2L.PolymerBehaviors.LocalizeBehaviorImpl = {
 	properties: {
 		formatDateTime: {
 			type: Function,
-			computed: '_computeFormatDateTime(language)'
+			computed: '_computeFormatDateTime(language, __resolveFast)'
 		},
 		formatDate: {
 			type: Function,
-			computed: '_computeFormatDate(language)'
+			computed: '_computeFormatDate(language, __resolveFast)'
 		},
 		formatFileSize: {
 			type: Function,
-			computed: '_computeFormatFileSize(language)'
+			computed: '_computeFormatFileSize(language, __resolveFast)'
 		},
 		formatNumber: {
 			type: Function,
-			computed: '_computeFormatNumber(language)'
+			computed: '_computeFormatNumber(language, __resolveFast)'
 		},
 		formatTime: {
 			type: Function,
-			computed: '_computeFormatTime(language)'
+			computed: '_computeFormatTime(language, __resolveFast)'
```

Comment:
Is this a pattern that we should be expected to follow in polymer in our teams? Why don't we just give language a default value of `'en-us'`?

Reply:
I wouldn't worry about repeating this pattern, it's a pretty strange case where we want functions to actually be computed properties like this. We only do it so that we can support re-rendering if the language on the page changes dynamically after render, which in practice doesn't happen outside of fun tests.